scripts/server.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+gunicorn --bind 0.0.0.0:8080 --workers 3 selectedtests.app.wsgi
```

Comment:
[nit] It might make these settings a little more manageable in the long run if we put these in a separate config file and passed that as an argument instead. 

http://docs.gunicorn.org/en/stable/configure.html#configuration-file

Reply:
There are only two settings right now, but if we add more I am happy to break it out into a config file